src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -40,70 +40,70 @@ internal static class PinvokeDllNames
-        internal const string RevertToSelfDllName = "api-ms-win-security-base-l1-2-0.dll";                   /*78*/
+        internal const string CreateNamedPipeDllName = "api-ms-win-core-namedpipe-l1-1-0.dll";               /*74*/
+        internal const string WaitNamedPipeDllName = "api-ms-win-core-namedpipe-l1-1-0.dll";                 /*75*/
+        internal const string PrivilegeCheckDllName = "api-ms-win-security-base-l1-1-0.dll";                 /*76*/
+        internal const string ImpersonateNamedPipeClientDllName = "api-ms-win-core-namedpipe-l1-1-0.dll";    /*77*/
+        internal const string RevertToSelfDllName = "api-ms-win-security-base-l1-1-0.dll";                   /*78*/
         internal const string CreateProcessInComputeSystemDllName = "vmcompute.dll";                         /*79*/
-        internal const string CLSIDFromProgIDDllName = "api-ms-win-core-com-l1-1-1.dll";                     /*80*/
-        internal const string LoadLibraryEx = "api-ms-win-core-libraryloader-l1-2-0.dll";                    /*81*/
-        internal const string FreeLibrary = "api-ms-win-core-libraryloader-l1-2-0.dll";                      /*82*/
+        internal const string CLSIDFromProgIDDllName = "api-ms-win-core-com-l1-1-0.dll";                     /*80*/
+        internal const string LoadLibraryEx = "api-ms-win-core-libraryloader-l1-1-0.dll";                    /*81*/
+        internal const string FreeLibrary = "api-ms-win-core-libraryloader-l1-1-0.dll";                      /*82*/
         internal const string EventActivityIdControlDllName = "api-ms-win-eventing-provider-l1-1-0.dll";     /*83*/
         internal const string GetConsoleCPDllName = "api-ms-win-core-console-l1-1-0.dll";                    /*84*/
         internal const string GetConsoleOutputCPDllName = "api-ms-win-core-console-l1-1-0.dll";              /*85*/
-        internal const string GetConsoleWindowDllName = "api-ms-win-core-kernel32-legacy-l1-1-1.dll";        /*86*/
-        internal const string GetDCDllName = "ext-ms-win-ntuser-dc-access-ext-l1-1-0.dll";                   /*87*/
-        internal const string ReleaseDCDllName = "ext-ms-win-ntuser-dc-access-ext-l1-1-0.dll";               /*88*/
-        internal const string TranslateCharsetInfoDllName = "ext-ms-win-gdi-font-l1-1-1.dll";                /*89*/
-        internal const string GetTextMetricsDllName = "ext-ms-win-gdi-font-l1-1-1.dll";                      /*90*/
-        internal const string GetCharWidth32DllName = "ext-ms-win-gdi-font-l1-1-1.dll";                      /*91*/
+        internal const string GetConsoleWindowDllName = "api-ms-win-core-kernel32-legacy-l1-1-0.dll";        /*86*/
+        internal const string GetDCDllName = "ext-ms-win-ntuser-dc-access-ext-l1-1-0.dll";                   /*87*/ // This DLL is not shipped as a downlevel forwarder, so we might want to revert to user32.dll
```

Comment:
I created issue #2091 to track the issue you brought up. It needs additional research, but should not block this PR to establish initial Win 7 support.